backoffice/src/Data/DatabaseSeeder.cs

```diff
@@ -258,4 +265,20 @@ private static void SeedNotes(DroitsContext dbContext)
 
         return notes;
     }
+
+
+    private static IEnumerable<Image> GetImages()
+    {
+        return Enumerable.Range(0, 1)
+            .Select(i => new Image()
+            {
+                Id = new Guid("d37319ba-5b77-45c1-87ee-551dc92fb8c4"),
```

Comment:
Gonna wanna either:

1. Keep a test image we can keep pointing to in the repo (probably have the key as something like test-image)
2. Upload one and use that key

I would probably side with the first option, but we don't want to have hardcoded "real" data here

Reply:
Id and Key changed to not be hard coded with a Guid. Should I rename file to be test image as well.